src/main/java/seedu/address/logic/commands/lesson/ListLessonCommand.java

```diff
@@ -0,0 +1,43 @@
+package seedu.address.logic.commands.lesson;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_LESSONS;
+
+import java.util.Optional;
+import java.util.function.Predicate;
+
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.model.Model;
+import seedu.address.model.commons.Code;
+import seedu.address.model.lesson.Lesson;
+
+public class ListLessonCommand extends Command {
+    public static final String COMMAND_WORD = "list";
+    public static final String MESSAGE_SUCCESS = "Listed all lessons";
+    // TODO: edit these messages
+    public static final String MESSAGE_USAGE = "L " + COMMAND_WORD + ": sample";
```

Comment:
Should MESSAGE_USAGE be `L list: sample`? Maybe change to another word like `CS2100` to make it clearer?